Homebrew/emacs_formula.rb

```diff
@@ -81,7 +81,7 @@ def ert_run_tests(*files)
 
   ### Slightly better tested than ert_run_tests
   def byte_compile(*files)
-    emacs_args = %W[ --batch -Q ]
```

Comment:
Batch mode will still load configuration files by default, which we definitely don't want.